forte/data/data_pack.py

```diff
@@ -907,6 +967,20 @@ def get_data(
                         a_type, a_args, data, context
                     )
 
+            if audio_annotation_types:
+                for a_type, a_args in audio_annotation_types.items():
+                    if a_type.__name__ in data.keys():
```

Comment:
For checking existence of key, just do 
```suggestion
                    if a_type.__name__ in data:
```